codelists/codelists.json

```diff
@@ -2,1107 +2,1113 @@
       "sha": "d5cfd421be9c680373365f60395a6a7b2d0ed82e"
     },
     "user-hjforbes-obsessive-compulsive-disorder-ocd.csv": {
       "id": "user/hjforbes/obsessive-compulsive-disorder-ocd/17792e81",
-      "url": "https://codelists.opensafely.org/codelist/user/hjforbes/obsessive-compulsive-disorder-ocd/17792e81/",
+      "url": "https://www.opencodelists.org/codelist/user/hjforbes/obsessive-compulsive-disorder-ocd/17792e81/",
       "downloaded_at": "2022-08-22 08:13:59.016226Z",
       "sha": "03a163b6afa744cf5ac8853eed58b2d798805e80"
     },
     "user-hjforbes-post-traumatic-stress-disorder.csv": {
       "id": "user/hjforbes/post-traumatic-stress-disorder/7e69bb4b",
-      "url": "https://codelists.opensafely.org/codelist/user/hjforbes/post-traumatic-stress-disorder/7e69bb4b/",
+      "url": "https://www.opencodelists.org/codelist/user/hjforbes/post-traumatic-stress-disorder/7e69bb4b/",
       "downloaded_at": "2022-08-22 08:13:59.372745Z",
       "sha": "d40c357ef9d0f171c9b7322911c6399b700e042f"
     },
     "user-hjforbes-diagnoses-eating-disorder.csv": {
       "id": "user/hjforbes/diagnoses-eating-disorder/62a78820",
-      "url": "https://codelists.opensafely.org/codelist/user/hjforbes/diagnoses-eating-disorder/62a78820/",
+      "url": "https://www.opencodelists.org/codelist/user/hjforbes/diagnoses-eating-disorder/62a78820/",
       "downloaded_at": "2022-08-22 08:13:59.726913Z",
       "sha": "7e6ed67c391b24de8f85adcca66813026596f67d"
     },
     "user-hjforbes-severe-mental-illness.csv": {
       "id": "user/hjforbes/severe-mental-illness/2dd5e1c0",
-      "url": "https://codelists.opensafely.org/codelist/user/hjforbes/severe-mental-illness/2dd5e1c0/",
+      "url": "https://www.opencodelists.org/codelist/user/hjforbes/severe-mental-illness/2dd5e1c0/",
       "downloaded_at": "2022-08-22 08:14:00.142054Z",
       "sha": "34556a280d7c7e90ce823c84930a5209f011ad0e"
     },
     "user-hjforbes-intentional-self-harm-aged10-years.csv": {
       "id": "user/hjforbes/intentional-self-harm-aged10-years/18951e5c",
-      "url": "https://codelists.opensafely.org/codelist/user/hjforbes/intentional-self-harm-aged10-years/18951e5c/",
+      "url": "https://www.opencodelists.org/codelist/user/hjforbes/intentional-self-harm-aged10-years/18951e5c/",
       "downloaded_at": "2022-08-22 08:14:00.631906Z",
       "sha": "1178e91131e6c33a05854dc0c80e5b1bc96d4604"
     },
     "user-hjforbes-undetermined-intent-self-harm-aged15-years.csv": {
       "id": "user/hjforbes/undetermined-intent-self-harm-aged15-years/4d66c4bc",
-      "url": "https://codelists.opensafely.org/codelist/user/hjforbes/undetermined-intent-self-harm-aged15-years/4d66c4bc/",
+      "url": "https://www.opencodelists.org/codelist/user/hjforbes/undetermined-intent-self-harm-aged15-years/4d66c4bc/",
       "downloaded_at": "2022-08-22 08:14:01.124763Z",
       "sha": "f1984cc22e5046b1f68319d8459e0715106768e0"
     },
     "user-hjforbes-suicide-icd-10.csv": {
```

Comment:
Please review the code lists you are loading. A large number of these code lists are not used in your study definition and should be removed.